sandbox/public/index.html

```diff
@@ -52,13 +61,16 @@
         prehidingStyle: ".personalization-container { opacity: 0 !important }",
         idSyncContainerId: 81,
         migrateIds: true,
-        onBeforeEventSend({ xdm, data }) {
-          const urlSearchParams = new URLSearchParams(window.location.search);
-          if (urlSearchParams.has("title")) {
-            xdm.web.webPageDetails.name = urlSearchParams.get("title");
+        onBeforeEventSend: function(options) {
+          const xdm = options.xdm;
+          const data = options.data;
+          const titleParam = getUrlParameter("title");
+          if (titleParam) {
+            xdm.web.webPageDetails.name = titleParam;
           }
-          if (urlSearchParams.has("campaign")) {
-            data.campaign = urlSearchParams.get("campaign");
+          const campaignParam = getUrlParameter("title");
+          if (campaignParam) {
+            data.campaign = campaignParam;
```

Comment:
I wanted to manually test the changes using the sandbox inside IE. In order to do that, I needed to convert this code to ES5.

Reply:
At what point was this breaking in IE?  During the build?  I was hoping this would throw an error when called, which is handled by the eventManager.

Reply:
At runtime. It probably was eventManager throwing the error, but it seems odd to just ignore the error in IE rather than fix it.

Reply:
I took another look. Initially IE was throwing an error when parsing index.html because of this syntax:

`onBeforeEventSend({ xdm, data }) {`

so I converted that line to ES5, which fixed that particular error. After that, I ran into the error that was logged by the eventManager due to URLSearchParams being undefined and decided to fix that as well. Interestingly, URLSearchParams is sometimes defined and sometimes is undefined by the time the `onBeforeEventSend` function runs. It looks like there's code being loaded asynchronously, probably from the create-react-app stuff, that provides a polyfill for URLSearchParams. Considering onBeforeEventSend fails some of the time due to URLSearchParams being undefined, I think it's best to just avoid the problem altogether.